queries/glimmer_javascript/highlights.scm

```diff
@@ -0,0 +1,65 @@
+; inherits: ecma
+
+(glimmer_opening_tag) @tag.builtin
+
+(glimmer_closing_tag) @tag.builtin
+
+;
+;
+; Copied from the javascript queries
```

Comment:
Why not simply inherit from Javascript?

Reply:
JavaScript includes JSX, which causes errors, because JSX is not valid is GJS files (I removed all the JSX nodes, too).